### DISCUSSION TITLE
Sylvania/Osram Lightify BR30 RGBW firmware

### DISCUSSION THREAD

**Discussion by BeerDiet** [Koenkk/zigbee2mqtt#13743] [Devices]
Hi.

Looking to upgrade the firmware on these.  Looking here:  https://www.osram.com/cb/lightify/lightify-home/lightify-home-faq/lightify_home_faqs.jsp

Expand the section highlighted below:

<img width="1001" alt="image" src="https://user-images.githubusercontent.com/9092387/187091414-ca775e3b-23f4-46e5-8744-2094032069dd.png">

Under it is a table.  To save you the trouble, go to Zigbee Device 38 (Manuf ID 4364, Image type 11).  The download link is http://s3-eu-west-1.amazonaws.com/lightify-osram-firmware/firmware/release/Cloud/NAFTA/NAFTA_BR_Rgbw_0x01020492_encrypt.bin

The questions I have... 

1. Is this image correct for an OTA update?  
2. It's not in the OTA index, is this because it was removed or just not put there yet?
3. Has anyone applied this update?

**Comment by zizzerzazzerzuzz** [comment]
I realize this is a little old, and the Devs may already know this, but encrypted firmware for sylvania, osram, lightify, ledvance (others?) exists here:  https://update.ledvance.com/firmware-overview
